metals/src/main/scala/scala/meta/internal/pantsbuild/commands/SharedCommand.scala

```diff
@@ -21,9 +21,13 @@ import java.nio.charset.StandardCharsets
 import bloop.bloopgun.core.Shell
 import scala.concurrent.Promise
 import scala.meta.internal.metals.BuildInfo
+import java.nio.file.Path
 
 object SharedCommand {
-  def interpretExport(export: Export): Int = {
+  def interpretExport(
+      export: Export,
+      coursierBinary: Option[Path] = None
```

Comment:
If we move `coursierBinary` into `ExportOptions` then we can avoid adding this parameter here and then the flag becomes available to all commands that trigger an export: create/refresh/amend